pkg/cmd/cli/cmd/project.go

```diff
@@ -66,7 +66,6 @@ func NewCmdProject(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.
 		Short:   "Switch to another project",
 		Long:    projectLong,
 		Example: fmt.Sprintf(projectExample, fullName),
-		Aliases: []string{"projects"},
```

Comment:
@kargakis we consider ok to use aliases for things like plural/singular. But aliases should not be "smart" so for for anything other than that (e.g. remove/delete) we must use suggestions like in `$ oc remove\n did you mean delete?`.